src/test/components/elements/EventBookingForm.test.tsx

```diff
@@ -178,34 +178,52 @@ describe("EventBookingForm", () => {
   it("shows an error message if first name is missing", () => {
     const targetUser = { ...mockUserSummary, givenName: undefined };
     setupTest({ role: "STUDENT", user: targetUser });
-    const formFeedback = screen.getByText("First name is required");
-    expect(formFeedback).toBeInTheDocument();
+    const firstName = screen.getByText("First name is required");
+    expect(firstName).toBeValid();
```

Comment:
Two things here; firstly you want to be selecting the text box itself, not the form feedback element. So `screen.getByRole("textbox", { name: /first name/i,})` or similar. Then you'll want to check that it's `.toBeInvalid()` because you would expect the textbox to be invalid if the name is missing. 

Currently you're still checking that the FormFeedback just exists.

Reply:
This has been removed